input/elasticapm/internal/modeldecoder/v2/model.go

```diff
@@ -979,19 +987,7 @@ type log struct {
 	Message nullable.String `json:"message"`
 	// FAAS holds fields related to Function as a Service events.
 	FAAS faas `json:"faas"`
-	// Labels are a flat mapping of user-defined key-value pairs.
-	Labels map[string]any `json:"labels" validate:"inputTypesVals=string;bool;number,maxLengthVals=1024"`
-
-	// Below embedded fields are added to enable supporting both nested and flat JSON.
-	// This is achieved by generating code using static analysis of these structs.
-	// The logic parses JSON tag of each struct field to produce a code which, at runtime,
-	// checks the nested map to retrieve the required value for each field.
-
-	EcsLogEventFields
-	EcsLogServiceFields
 	EcsLogLogFields
```

Comment:
Maybe also move this one up to the other embedded fields for readability, or add another comment.